.github/workflows/cypress_workflow.yml

```diff
@@ -179,8 +198,6 @@ jobs:
 
             #### Inputs:
             ```
-            Source repo: '${{ env.SOURCE_REPO }}'
```

Comment:
Got rid of this because unless set at a file level github env variables do not share between jobs. This information is already available within the test run.